src/core.c/IO/Path.pm6

```diff
@@ -766,92 +779,122 @@ my class IO::Path is Cool does IO {
         nqp::hllbool(Rakudo::Internals.FILETEST-E(self.absolute))
```

Comment:
Missed this one

Reply:
Ha. I did look at it, but changing to use syscalls won't reduce the number of `stat`s called, so I left it as-is. `Rakudo::Internals.FILETEST-E` does just one `nqp::stat`. If we want to completely remove all uses of `nqp::stat`, I'll make the change, but will then need to make more changes elsewhere.